Benchmarks/shootout/binarytrees/binarytrees.r

```diff
@@ -45,6 +45,6 @@ binarytrees <- function(args) {
         check(long_lived_tree), "\n")
 }
 
-execute <- function(n) {
+execute <- function(n=10) {

```

Comment:
Why the default arg? Because our benchs run faster with it? The value for each iteration in the rebench file was calculated so that each bench iteration takes around 1s in the benchmarking machine. That number should be adapted for each new environment in which the benchs run. I do not think it is a good idea to use that number as the default for each benchmark.